### ISSUE TITLE
clusters plotter is incompatible with pytorch on windows because of 'control' library

### ISSUE THREAD

**Issue by haesleinhuepf** [BiAPoL/napari-clusters-plotter#248]
When installing napari-clusters-plotter and pytorch in the same conda environment, pytorch doesn't work unfortunately. The problem is that the library `control` breaks `pytorch`. `control` is installed because the clusters-plotter installs `finesse`, which depends on `control`. The issue only comes when installing the clusters-plotter using mamba or conda. The pip installation is fine.

[finesse]() is listed under [requirements on conda-forge](https://github.com/conda-forge/napari-clusters-plotter-feedstock/blob/6c4c3f661b2b3007b087b6bd75427aee5d66ac31/recipe/meta.yaml#LL34C7-L34C14), but not in [requirements.txt](https://github.com/BiAPoL/napari-clusters-plotter/blob/main/requirements.txt) or [setup.py](https://github.com/BiAPoL/napari-clusters-plotter/blob/main/setup.cfg). I can confirm that installing it via pip doesn't cause the issue.

To reproduce set up an environment like this:
```
mamba create --name test10 python=3.9 napari-clusters-plotter -c conda-forge

mamba activate test10

mamba install pytorch torchvision torchaudio pytorch-cuda=11.7 -c pytorch -c nvidia
```

Run this python script:
```
import torch
```
Error:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\haase\mambaforge\envs\test10\lib\site-packages\torch\__init__.py", line 122, in <module>
    raise err
OSError: [WinError 182] The operating system cannot run %1. Error loading "C:\Users\haase\mambaforge\envs\test10\lib\site-packages\torch\lib\nvfuser_codegen.dll" or one of its dependencies.
```

## Figuring out dependencies
I used `conda-tree` to figure that out:
```
mamba create --name bio393 python=3.9 devbio-napari -c conda-forge
mamba activate bio393
mamba install -c conda-forge conda-tree
conda-tree whoneeds control
```
> finesse
```
conda-tree whoneeds finesse
```
> napari-clusters-plotter

## See also
* https://github.com/python-control/python-control/issues/815
* https://github.com/pytorch/pytorch/issues/89752

**Comment by haesleinhuepf** [comment]
I will make a new release on pypi and conda-forge to remove the `finesse` dependency. In case anybody thinks `finesse` should stay, please report issues.

**Comment by Cryaaa** [comment]
I don't even know that library and since it is not in the pip dependencies I'm fine with removing it

**Comment by Cryaaa** [comment]
Fixed in #274
